code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -154,14 +154,15 @@
 	else if(istype(W, /obj/item/stack/packageWrap))
 		return
 	else if(istype(W, /obj/item/stack/cable_coil))
+		var/obj/item/stack/cable_coil/C = W
 		if(rigged)
 			to_chat(user, "<span class='notice'>[src] is already rigged!</span>")
 			return
-		to_chat(user, "<span class='notice'>You rig [src].</span>")
-		user.drop_item()
-		qdel(W)
-		rigged = 1
-		return
+		if(C.use(15))
+			to_chat(user, "<span class='notice'>You rig [src].</span>")
+			rigged = 1
+		else 
+			to_chat(user, "<span class='warning'>You need atleast 15 wires to rig [src]!</span>")

```

Comment:
should add a return after this else statement

Reply:
In this case it isn't needed due to the else if structure. But I'll include it to be more readable